src/components/Header.js

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React from 'react';
+import styled from 'styled-components/macro';
+
+import linkedInIcon from 'assets/linkedin-color-icon-white.svg'
+import gitHubIcon from 'assets/github-icon-white.svg'
+import stackOverflowIcon from 'assets/stackoverflow-icon-white.svg'
+import profilePic from 'assets/profile-pic.jpg'
+
+import { gitHubLink, linkedInLink, stackOverflowLink } from 'utils/Urls';
+import { OuterWrapper, InnerWrapper } from 'styles/ProjectStyles';
+
+const Header = () => {
+  const socialLinksHeader = [
+    {
+      name: 'Linkedin',
+      link: linkedInLink,
+      image: linkedInIcon
+    },
+    {
+      name: 'Github',
+      link: gitHubLink,
+      image: gitHubIcon
+    },
+    {
+      name: 'Stackoverflow',
+      link: stackOverflowLink,
+      image: stackOverflowIcon
+    }
+  ]
```

Comment:
I did the same with these, also in the MoreInfo - like you did also. But I was thinking it should be a way to do it once and re-use.